src/peft/utils/peft_types.py

```diff
@@ -57,6 +58,7 @@ class PeftType(str, enum.Enum):
     OFT = "OFT"
     POLY = "POLY"
     LN_TUNING = "LN_TUNING"
+    LORAPLUS = "LORAPLUS"
```

Comment:
I don't think it's needed here, we will never initialize a "LORAPLUS" PEFT model.

Reply:
The tests rely on its existence in the "try all the peft types" test(s).